### DISCUSSION TITLE
Dragging cues...

### DISCUSSION THREAD

**Discussion by emccarron** [FrancescoCeruti/linux-show-player#332] [Q&A] [answered]
When I clone a cue, it doesn't want to scroll when I drag past the bottom -- I have to drag and drop multiple times?

I figure I'm doing something wrong.

Alternatively, can "clone cue" add at the end of the cue list, instead of immediately after the clopned cue?  Is there a kybd shortcut for that I'm missing?

**Comment by emccarron** [comment]
OK, so resolved that with presets.

Any way to make intermediate cues?  Like 3.1, 3.2 as the director changes their mind?

**Reply by FrancescoCeruti** [reply]
The "index" currently doesn't allow this, but you can hide that column and have the numbers in the cue name

**Reply by emccarron** [reply]
This was one of those "Why didn't I see that" moments.  Much easier.  Thanks!

**Reply by emccarron** [reply]
Now... Got any tricks to keep the director from changing her mind? :)

**Reply by FrancescoCeruti** [reply]
We can only dream :laughing:

**Comment by FrancescoCeruti** [comment, marked as answer]
Hi! You should be able to make the view scroll while dragging to the bottom, it's a bit tricky, when I'm back at my computer I can make a screen capture.

**Reply by FrancescoCeruti** [reply]
https://github.com/user-attachments/assets/840bfdbf-e207-4640-89cb-16ed3485e88a

**Comment by emccarron** [comment]
Heh, I was using the touchpad and was totally not coordinated enough.  Works much better with the mouse.

Thanks!
